src/view/shell/index.tsx

```diff
@@ -92,6 +94,7 @@ function ShellInner() {
   }, [dedupe, navigation])
 
   const swipeEnabled = !canGoBack && hasSession && !isDrawerSwipeDisabled
+  const [trendingScrollGesture] = useState(() => Gesture.Native())
```

Comment:
It's annoying we have to hoist it way up here. I think we could've used `blocksExternalGesture` instead of `requireExternalGestureToFail` (and thus reverse the relationship) but it seems broken, alas: https://github.com/software-mansion/react-native-gesture-handler/issues/3321